src/arkit.cs

```diff
@@ -856,20 +976,37 @@ interface ARFaceGeometry : NSCopying, NSSecureCoding {
 	[BaseType (typeof(SCNGeometry))]
 	[DisableDefaultCtor]
 	interface ARSCNFaceGeometry {
+		[Obsolete ("Use the 'Create' static constructor instead.")]
```

Comment:
put `Obsolete` under a `!XAMCORE_4_0`
also invert them, so the right one (this one) does not call the obsolete one
i.e. the _bad_ one should suffer the extra call (and makes the `!XAMCORE_4_0` compile

Reply:
Makes sense